### PR TITLE
feat: Phone number tab sign in UI with internationalization

### PR DESCRIPTION
**Pull request by TikuGaelle**
## Description

* Sign in page phone number tab UI. Implementation of issue #67 
* Internationalization of text in both french and English

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read and ran all relevant commands as specififed in the Running Tests section of the [Contributor Guide].
- [ ] The title of the PR follows the [Conventional Commits] guideline
- [ ] My local branch follows the naming standards in the [Deepsource Branch Naming Convention] or [Biodiversity Branch Naming Convention]
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy],
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

[Contributor Guide]: https://github.com/FlutterPlaza/.github/blob/main/CONTRIBUTING.md
[Conventional Commits]: https://www.conventionalcommits.org/en/v1.0.0-beta.4/
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[Biodiversity Branch Naming Convention]: https://bit.ly/3DyYSwM
[Deepsource Branch Naming Convention]: https://bit.ly/3Y08Gs4

### CODE REVIEW

**Review by fonkamloic — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts first

File: lib/sign_in/view/sign_in_page.dart [hunk 1]
```diff
@@ -167,7 +168,49 @@ class _SignInBodyState extends State<SignInBody>
...
+                                                    cts.maxHeight * 0.025,
+                                                  ),
+                                                ),
+                                              ),
+                                            ),
+                                            initialCountryCode: 'US',
+                                          ),
+                                        ],
+                                      ),
+                                      const PasswordInput(),
```
Comment (fonkamloic) [inline lib/sign_in/view/sign_in_page.dart]: Export the PhoneInput as a component of widget the same way it is done with this passwordInput

**Comment by fonkamloic** [comment]
Screenshot 2023-02-11 at 3 03 02 PM

let the implementation match the design

**Review by lkmandy — CHANGES_REQUESTED** [review]
The UI currently looks like this on my end



Can you confirm that the UI displays properly on your end?

**Comment by TikuGaelle** [comment]
> 



> The UI currently looks like this on my end
> 
> 
> 
> Can you confirm that the UI displays properly on your end?

Please run again now.

**Comment by lkmandy** [comment]
@TikuGaelle 

The phone number section is missing and the UI is still alittle crunched at the level of the email title

File: lib/core/presentation/theming/themes/light_theme.dart [hunk 2]
```diff
@@ -51,6 +50,7 @@ ThemeData whiteTheme(BuildContext context, BoxConstraints cts) {
       ),
       titleSmall: style.titleSmall?.copyWith(
         fontSize: cts.maxWidth * 0.04,
+        color: Color(0xff808191)
```
Comment (fonkamloic) [inline lib/core/presentation/theming/themes/light_theme.dart]: No static colors in file. Colors will need to switch based on theme brightness. light or dark

Comment (fonkamloic) [reply lib/core/presentation/theming/themes/light_theme.dart]: define colors as part of colors file.

File: lib/core/presentation/widget/my_button.dart [hunk 3]
```diff
@@ -30,7 +30,7 @@ class FpbButton extends StatelessWidget {
     final size = MediaQuery.of(context).size;
     return SizedBox(
       width: width ?? size.width,
-      height: height ?? size.height * 0.075,
+      height: height ?? size.height * 0.065,
```
Comment (fonkamloic) [inline lib/core/presentation/widget/my_button.dart]: We have to change these values based on Screen dimensions.
small
small
large
xlarge
medium.

File: lib/onboarding/view/widgets/alternative_auth.dart [hunk 4]
```diff
@@ -14,32 +14,34 @@ class AlternativeAuth extends StatelessWidget {
...
-        ),
-        IconLogin(
-          svg: SvgNames.apple,
-          box: box,
-          onTap: () {},
-        ),
-      ],
-    );
+    return Wrap(direction: Axis.horizontal, children: [
+      Row(
```
Comment (fonkamloic) [inline lib/onboarding/view/widgets/alternative_auth.dart]: Why the wrap?

File: lib/sign_in/view/sign_in_page.dart [hunk 5]
```diff
@@ -90,131 +91,160 @@ class _SignInBodyState extends State<SignInBody>
...
                   Align(
                     alignment: Alignment.bottomCenter,
-                    child: Column(
-                      children: [
-                        Row(
-                          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    //heightFactor: cts.maxHeight * .75,
+                    child: Container(
+                      height: cts.maxHeight * .8,
+                      child: ListView(children: [
```
Comment (fonkamloic) [inline lib/sign_in/view/sign_in_page.dart]: Why listView?

**Review by fonkamloic — CHANGES_REQUESTED** [review]
Please just include the following files in PR
1- phone_number_input.dart
2- *.arb files
3- pubspec.yaml 

The other files will create a regression.

**Comment by TikuGaelle** [comment]
> Please just include the following files in PR 1- phone_number_input.dart 2- *.arb files 3- pubspec.yaml
> 
> The other files will create a regression.

Okay.

File: lib/sign_in/view/sign_in_page.dart (hunk 5 above)
Comment (TikuGaelle) [reply lib/sign_in/view/sign_in_page.dart]: For Scrolling effect, which makes the design more responsive on all screen sizes...

File: lib/onboarding/view/widgets/alternative_auth.dart (hunk 4 above)
Comment (TikuGaelle) [reply lib/onboarding/view/widgets/alternative_auth.dart]: Responsiveness to screen sizes of lesser width, which will make the icons move to the next line if it overflows

File: lib/core/presentation/widget/my_button.dart (hunk 3 above)
Comment (TikuGaelle) [reply lib/core/presentation/widget/my_button.dart]: Okay.

File: lib/core/presentation/theming/themes/light_theme.dart (hunk 2 above)
Comment (TikuGaelle) [reply lib/core/presentation/theming/themes/light_theme.dart]: Okay.

**Review by fonkamloic — APPROVED** [review]
LGTM

**Pull request merged**